app/server.go

```diff
@@ -694,7 +694,7 @@ func (s *Server) initLogging() error {
 
 		listenerId := cfg.AddListener(func(_, newCfg mlog.LogTargetCfg) {
 			if err := s.Log.ConfigAdvancedLogging(newCfg); err != nil {
-				mlog.Error("Error re-configuring advanced logging", mlog.Err(err))
+				mlog.Warn("Error re-configuring advanced logging", mlog.Err(err))
```

Comment:
An admin would be the one to change configuration... wouldn't we want the admin to see it didn't work?  An example error would be a typo in the new config.

Reply:
Warning messages are still observable, I think admins can still see this error. But I agree that it may also makes sense to keep this at error level. 0/5 on keeping this at warning level.